### ISSUE TITLE
Readme

### ISSUE THREAD

**Issue by Jokers-Exchange** [Kafium/Kafium#5]
You should add more information about the blockchain in the readme file, like that you have in store for the future, where you are in the project currently, how's it going to be different than other blockchains, and more information like that.

**Comment by Lennster1** [comment]
I agree, first we should put forward some main selling points of kafium. Like its greener, but how? etc;

**Comment by KaffinPX** [comment]
> I agree, first we should put forward some main selling points of kafium. Like its greener, but how? etc;

We dont need selling points :) Ill add some functionalities.
